weekly-scan/template.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "weekly-scan-job-template"
+objects:
+  - apiVersion: "v1"
+    kind: "BuildConfig"
+    metadata:
+      name: ${PIPELINE_NAME}
+    spec:
+      source:
```

Comment:
We don't need source of the project here. We can remove the `source` altogether.